src/RpcServer/RpcError.cs

```diff
@@ -16,12 +16,22 @@ public class RpcError
     {
         #region Default Values
 
-        //public static readonly RpcError InternalServerError = new(-32603, "Internal server RpcError");
-        public static readonly RpcError BadRequest = new(-32700, "Bad request");
+        // https://www.jsonrpc.org/specification
+        // | code               | message         | meaning                                                                           |
+        // |--------------------|-----------------|-----------------------------------------------------------------------------------|
+        // | -32700             | Parse error     | Invalid JSON was received by the server. An error occurred on the server while parsing the JSON text. |
+        // | -32600             | Invalid Request | The JSON sent is not a valid Request object.                                      |
```

Comment:
s/`Invalid Request`/`Invalid request`

Reply:
I think this would be fine, i coped it directly from the official website